src/org/zaproxy/admin/resources/help/contents/addon.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
```

Comment:
As mentioned in zaproxy/zap-extensions#610, we should set the appropriate charset (`<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">`).